### DISCUSSION TITLE
-theme-str causes an error if ran through sxhkd but not from the terminal

### DISCUSSION THREAD

**Discussion by promitheas17j** [davatorium/rofi#1972] [Theming] [answered]
I am having a small issue while trying to run rofi through an sxhkd keybind. The exact command I am using works if I run it directly in the terminal.

Here it is:
```
rofi -dmenu -i -markup-rows -no-show-icons -input /home/user/.keybinds_list -theme-str 'window {width: 80%;}' -p "Search Keybinds"
```
I get an error message in a window without my theme applied:

> Error while parsing theme: window width: 80%;
> Parser error: syntax error, unexpected invalid element name, expecting "bracket open ('{')" or "Selector separator (',')"

If I run it from the terminal directly i will get a properly sized window with the correct contents from the keybinds_list file. Also, if I remove the -theme-str option it will again run, but its not the right size that I want (obviously).

My rofi version is: 1.7.5

**Comment by DaveDavenport** [comment, marked as answer]
It looks like sxhkd is messing up your command. (removing the '{}') 

Yes, you need to escape '{}' : https://wiki.archlinux.org/title/Sxhkd

**Reply by promitheas17j** [reply]
Yes this was it. Thanks for your quick response!
